pkg/cmd/openshift/openshift.go

```diff
@@ -147,43 +146,7 @@ func newExperimentalCommand(name, fullName string) *cobra.Command {
 	return experimental
```

Comment:
These changes will not compile until https://github.com/openshift/origin/pull/17486 (which moves experimental commands into pkg/oc) is merged. This function contains the last use of `pkg/oc/cli/util/clientcmd` outside of the `pkg/oc` subtree.